src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Filter.cs

```diff
@@ -58,6 +58,16 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
 
             var fArgsValid = base.CheckTypes(context, args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
 
+            // Only a small percentage of makers use the multi-argument version of Filter in Canvas.
+            // The C# interpreter never supported it.  Blocking this and pushing makers to the more readable
+            // use of "And" to tie multiple predicates together, consistent with LookUp, and freeing up the 
+            // third argument to perform an efficient projection as is also done for LookUp.
+            if (args.Length > 2 && context.Features.PowerFxV1CompatibilityRules)
```

Comment:
If we plan to backcompat this change into Power Apps, we should add a new feature instead of using this one

Reply:
We talked and because there is a relatively large number of usres of the old form, we should avoid the back compat converter and leave it under V1 compatibility flag.